taxonomies/additives.txt

```diff
@@ -3466,7 +3611,19 @@ additives_classes:en: en:preservative
 vegan:en:yes
 vegetarian:en:yes
 
+<en:E202
+en:preservative potassium sorbate, preservative E202
```

Comment:
This should be removed, it will be handled by the parser.

Reply:
These are now often found as synonyms as well.